packages/tracing-enriched-data-handler/src/services/enrichedService.ts

```diff
@@ -0,0 +1,25 @@
+import { BucketService } from "./bucketService.js";
+import { DBService } from "./db/dbService.js";
+import { ProducerService } from "./producerService.js";
+
+export const enrichedServiceBuilder = (
+  dbService: DBService,
+  bucketService: BucketService,
+  producerService: ProducerService,
+) => {
+  return {
+    async insertTracing(message: unknown): Promise<unknown> {
+      const s3KeyPath = message as string;
```

Comment:
not sure if it is only for the scaffolding, but let's avoid casting when possible